resources/views/components/card/blog.blade.php

```diff
@@ -0,0 +1,8 @@
+<a href="{{ route('blogs.show', ['blog' => $blog]) }}">
+    <div class="box">
+        <h3>{{ $blog->title }}</h3>
+        <p>{!! $blog->excerpt !!}</p>
+        <h6>{{ $blog->created_at->toFormattedDateString() }}</h6>
+    </div>
+</a>
+<br>
```

Comment:
Adding a `<br>` tag is not the proper way to add some vertical spacing. Consider using a padding-bottom or something similar for that.